### ISSUE TITLE
Add CardView to supported background checking views 

### ISSUE THREAD

**Issue by HugoCastelani** [afollestad/aesthetic#8]
Using them with layouts is OK, but I could not with cardview. They should not return the same color despite the view?

**Comment by afollestad** [comment]
I'll add `CardView` to the supported views. Unfortunately I cannot easily support every view.

**Comment by HugoCastelani** [comment]
Actually, I was trying to change another view's color (a library, in fact), cardview was just an example, but if you got to do this manually and isn't an error, it doesn't matter. Thank you.

**Comment by afollestad** [comment]
CardView is commonly used so I think it would be good for me to add :)

**Comment by HugoCastelani** [comment]
For sure, but this is a really significant limitation. Are you considering adding a global support?

**Comment by afollestad** [comment]
The thing is that each individual view type subscribes to changes and manage their theme life cycle, which is part of what makes the library smaller and more efficient. This also allows them to get updates without recreating Activities.

**Comment by afollestad** [comment]
I'll try to think of a good way to globally support background colors though.

**Comment by HugoCastelani** [comment]
Awesome! Last question: I could not get primary/accent colors programmaticaly by Aesthetic class or by TypedValue method. Is it even possible?

**Comment by afollestad** [comment]
There are methods to retrieve your currently set values, for example:

```java
Aesthetic.get()
   .primaryColor()
   .subscribe(color -> {
        // Use color
    });
```

This is using an Observable from RxJava, you can make the subscription blocking so you can get the result without a callback. By default, primary color would return `colorPrimary` from your Activity theme.

**Comment by afollestad** [comment]
Note that `.take(1)` above waits for the next value and unsubscribes, so you don't continue to get updates.

**Comment by HugoCastelani** [comment]
I've never used Observable, so I didn't get its function at first sight. Thank you for explaining.

**Comment by afollestad** [comment]
Ah, no problem! I'll have to add some documentation for that.

**Comment by afollestad** [comment]
Correct support for `CardView` is available in 0.0.6
